cmd/internal/kube/daemonset/daemonset.go

```diff
@@ -143,12 +151,9 @@ func (d *Daemonset) Run() error {
 
 		// Continue until an interrupt
 	DoneWaitingForSignal:
-		for {
-			select {
-			case received := <-sig:
-				printer.Stderr.Infof("Received %v, stopping daemonset...\n", received.String())
-				break DoneWaitingForSignal
-			}
+		for received := range sig {
+			printer.Stderr.Infof("Received %v, stopping daemonset...\n", received.String())
+			break DoneWaitingForSignal
```

Comment:
Not sure what's going on here, but if you don't have the nested `for`/`select` then you don't need the named `break`